File: fizz_buzz.js
for (let i = 1; i <= 100; i++) {
    let v = ""
    if (i % 3 == 0 && i % 5 == 0) {
        v = "FizzBuzz";
        console.log(v);
    } else if (i % 3 == 0) {
        v = "Fizz";
        console.log(v);
    } else if (i % 5 == 0) {
        v = "Buzz";
        console.log(v);
    }
    else {
        v = i;
        console.log(v)
    }
}

File: fizz_buzz.py
for i in range(1, 101):
    v = ""
    if i % 3 == 0 and i % 5 == 0:
        v = "FizzBuzz"
        print(v)
    elif i % 3 == 0:
        v = "Fizz"
        print(v)
    elif i % 5 == 0:
        v = "Buzz"
        print(v)
    else:
        print(i)